### PR TITLE
Lazy prop on VExpansionPanelContent

### PR DESCRIPTION
**Pull request by jacekkarczmarczyk**
Playground.vue

```vue
<template>
  <v-app id="inspire">
    <v-expansion-panel>
    <v-expansion-panel-content v-for="(item,i) in 5" :key="i" :lazy="i%2 === 0">
      <div slot="header">Item</div>
      <v-card>
        <v-card-text class="grey lighten-3">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</v-card-text>
      </v-card>
    </v-expansion-panel-content>
  </v-expansion-panel>
  </v-app>
</template>

<script>
export default {
}
</script>
```

### CODE REVIEW

File: src/components/VExpansionPanel/VExpansionPanelContent.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { VExpandTransition } from '../transitions'
 
 import Toggleable from '../../mixins/toggleable'
+import Bootable from '../../mixins/bootable'
```
Comment (johnleider) [inline src/components/VExpansionPanel/VExpansionPanelContent.js]: move up (alphabetical)

File: src/components/VExpansionPanel/VExpansionPanelContent.js [hunk 2]
```diff
@@ -10,7 +11,7 @@ import ClickOutside from '../../directives/click-outside'
 export default {
   name: 'v-expansion-panel-content',
 
-  mixins: [Toggleable],
+  mixins: [Toggleable, Bootable],
```
Comment (johnleider) [inline src/components/VExpansionPanel/VExpansionPanelContent.js]: move to start of array (alphabetical)

File: src/components/VExpansionPanel/VExpansionPanelContent.js [hunk 3]
```diff
@@ -31,7 +32,8 @@ export default {
 
   props: {
     hideActions: Boolean,
-    ripple: Boolean
+    ripple: Boolean,
+    lazy: Boolean
```
Comment (johnleider) [inline src/components/VExpansionPanel/VExpansionPanelContent.js]: move up (alphabetical)

File: src/components/VExpansionPanel/VExpansionPanelContent.spec.js [hunk 4]
```diff
@@ -20,4 +20,14 @@ test('VExpansionPanelContent.js', ({ mount }) => {
 
   //   expect(wrapper.html()).toMatchSnapshot()
   })
+
+  it('should render an expanded component with lazy prop and match snapshot', () => {
```
Comment (johnleider) [inline src/components/VExpansionPanel/VExpansionPanelContent.spec.js]: is this meant to be commented out?

Comment (jacekkarczmarczyk) [reply src/components/VExpansionPanel/VExpansionPanelContent.spec.js]: Yes, similar to other tests for this components. It uses "inject" which as I understand is not supported yet in avoriaz

File: src/components/VExpansionPanel/VExpansionPanelContent.js (hunk 1 above)
Comment (jacekkarczmarczyk) [reply src/components/VExpansionPanel/VExpansionPanelContent.js]: PR updated

**Pull request merged**